pkg/ebpf/tracee.go

```diff
@@ -264,6 +274,11 @@ func New(cfg Config) (*Tracee, error) {
 		events:        GetEssentialEventsList(),
 	}
 
+	// Start capabilities rings
+	t.capabilities = capabilities.NewCapabilities(t.config.Capabilities.BypassCaps)
+	t.capabilities.Unprivileged() // move to ring3 right away
```

Comment:
Maybe this should just be the default behavior, so `NewCapabilities` returns an unprivileged `capabilities` object. Then it is a defined behavior that privileged actions is only opt in inside their scope and will not occur anywhere else.

Reply:
True. This is a leftover from my last change (adding scopes) so.. having "Unprivileged" by default is definitly the way to go.